packages/rancher-istio/charts/values.yaml

```diff
@@ -16,11 +19,11 @@ kiali:
     contextPath: /
     dashboard:
       jaegerInClusterURL: http://tracing.istio-system:80
-      # grafanaInClusterURL: http://access-grafana.cattle-prometheus:80
+      grafanaInClusterURL: http://access-grafana.cattle-prometheus:80
       auth:
         strategy: anonymous
     enabled: true
-    # prometheusAddr: http://access-prometheus.cattle-prometheus:80
+    prometheusAddr: http://access-prometheus.cattle-prometheus:80
```

Comment:
is grafana being removed ?

Reply:
Yes, as part of getting rid of as much as possible from the base overlay file and since its no longer an addon